modules/core/shared/src/main/scala/scaladex/core/model/Artifact.scala

```diff
@@ -221,29 +221,65 @@ case class Artifact(
     }
 
   // todo: Add tests for this
-  def scastieURL: String = {
-    val tryBaseUrl = "https://scastie.scala-lang.org/try"
+  // def scastieURL: String = {
+  //   val tryBaseUrl = "https://scastie.scala-lang.org/try"
+
+  //   val targetParam = binaryVersion.platform match {
+  //     case ScalaJs(_) => Some("t" -> "JS")
+  //     case _          => None
+  //   }
+
+  //   val scalaVersionParam = binaryVersion.language match {
+  //     case Scala(v) => Some("sv" -> v.toString)
+  //     case _        => None
+  //   }
+
+  //   val params: List[(String, String)] = List(
+  //     "g" -> groupId.value,
+  //     "a" -> artifactName.value,
+  //     "v" -> version.toString
+  //   ) ++ targetParam ++ scalaVersionParam
+
+  //   params
+  //     .map { case (k, v) => s"$k=$v" }
+  //     .mkString(tryBaseUrl + "?", "&", "")
+  // }
+
+  def isSbtPlugin: Boolean = platform match {
+    case SbtPlugin(_) => true
+    case _            => false
+  }
 
-    val targetParam = binaryVersion.platform match {
-      case ScalaJs(_) => Some("t" -> "JS")
-      case _          => None
-    }
+  def getOrganization: String = projectRef.organization.toString
 
-    val scalaVersionParam = binaryVersion.language match {
-      case Scala(v) => Some("sv" -> v.toString)
-      case _        => None
-    }
+  def getRepository: String = projectRef.repository.toString
 
-    val params: List[(String, String)] = List(
-      "g" -> groupId.value,
-      "a" -> artifactName.value,
-      "v" -> version.toString
-    ) ++ targetParam ++ scalaVersionParam
+  def scastieURL: Option[String] =
+    if (isSbtPlugin) {
```

Comment:
Also if it is Scala Native it should return `None`.

Reply:
Done, it will be visible in next commit, here's how I changed it to also return None if Scala Native- 

`if (isSbtPlugin || binaryVersion.platform == ScalaNative(_)) {
      None
    }` 

I hope this fulfills the requirements